freestyle-cache/shared/src/main/scala/cache.scala

```diff
@@ -20,6 +20,12 @@ package cache {
       // Sets the value of a key to a newValue.
       def put(key: Key, newVal: Val): FreeS.Par[F, Unit]
 
+      // Copy all of the mappings from the specified map to this map
```

Comment:
Should we say `to this cache` instead of `to this map`?

Similarly for `isEmpty` below (`this map contains` vs `this map contains`)?